File: conflictexample/src/main/java/com/donfyy/MainActivity.java
package com.donfyy;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.donfyy.c_scroll.case1.Case1Activity;
import com.donfyy.c_scroll.case2.Case2Activity;
import com.donfyy.c_scroll.case3.Case3Activity;
import com.donfyy.conflict.R;
import com.donfyy.willremove.intercept.innerintercept.InnerInterceptActivity;
import com.donfyy.d_innerouterconflict.outerinterceptlifthand.LiftHandActivity;
import com.donfyy.d_innerouterconflict.innerinterceptnolifthand.NoLiftHandActivity;
import com.donfyy.willremove.intercept.outerintercept.OuterInterceptActivity;
import com.donfyy.a_conflictdemo.ConflictActivity;
import com.donfyy.e_conflictfixcases.nestedscroll.NestedViewPagerActivity;
import com.donfyy.a_now.HeadlineNewsActivity;
import com.donfyy.b_viewarchitecture.TouchEventDemoActivity;
import com.donfyy.e_conflictfixcases.webview.WebviewActivity;
import com.donfyy.e_conflictfixcases.x5webview.X5WebviewActivity;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_home);
    }

    public void onButtonClick(View v) {
        if (v.getId() == R.id.nested_scroll) {
            Intent intent = new Intent(this, NestedViewPagerActivity.class);
            startActivity(intent);
        }  else if (v.getId() == R.id.case1) {
            Intent intent = new Intent(this, Case1Activity.class);
            startActivity(intent);
        }else if (v.getId() == R.id.case2) {
            Intent intent = new Intent(this, Case2Activity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.case3) {
            Intent intent = new Intent(this, Case3Activity.class);
            startActivity(intent);
        } /*else if (v.getId() == R.id.two_scroll) {
            Intent intent = new Intent(this, TwoScrollActivity.class);
            startActivity(intent);
        } */else if (v.getId() == R.id.nonlift) {
            Intent intent = new Intent(this, NoLiftHandActivity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.lift) {
            Intent intent = new Intent(this, LiftHandActivity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.conflict) {
            Intent intent = new Intent(this, ConflictActivity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.viewpager) {
            Intent intent = new Intent(this, HeadlineNewsActivity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.event_dispatch) {
            Intent intent = new Intent(this, TouchEventDemoActivity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.outer_intercept) {
            Intent intent = new Intent(this, OuterInterceptActivity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.inner_intercept) {
            Intent intent = new Intent(this, InnerInterceptActivity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.webview_conflict) {
            Intent intent = new Intent(this, WebviewActivity.class);
            startActivity(intent);
        } else if (v.getId() == R.id.x5webview_conflict) {
            Intent intent = new Intent(this, X5WebviewActivity.class);
            startActivity(intent);
        }
    }
}

File: conflict/src/main/java/com/donfyy/MainActivity.kt
package com.donfyy

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.view.View
import com.donfyy.a_conflictdemo.ConflictActivity
import com.donfyy.a_now.HeadlineNewsActivity
import com.donfyy.b_viewarchitecture.TouchEventDemoActivity
import com.donfyy.c_scroll.case1.Case1Activity
import com.donfyy.c_scroll.case2.Case2Activity
import com.donfyy.c_scroll.case3.Case3Activity
import com.donfyy.conflict.R
import com.donfyy.d_innerouterconflict.innerinterceptnolifthand.NoLiftHandActivity
import com.donfyy.d_innerouterconflict.outerinterceptlifthand.LiftHandActivity
import com.donfyy.e_conflictfixcases.nestedscroll.NestedViewPagerActivity
import com.donfyy.e_conflictfixcases.webview.WebviewActivity
import com.donfyy.e_conflictfixcases.x5webview.X5WebviewActivity
import com.donfyy.willremove.intercept.innerintercept.InnerInterceptActivity
import com.donfyy.willremove.intercept.outerintercept.OuterInterceptActivity

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_home)
    }

    fun onButtonClick(v: View) {
        when (v.id) {
            R.id.nested_scroll -> {
                val intent = Intent(this, NestedViewPagerActivity::class.java)
                startActivity(intent)
            }
            R.id.case1 -> {
                val intent = Intent(this, Case1Activity::class.java)
                startActivity(intent)
            }
            R.id.case2 -> {
                val intent = Intent(this, Case2Activity::class.java)
                startActivity(intent)
            }
            R.id.case3 -> {
                val intent = Intent(this, Case3Activity::class.java)
                startActivity(intent)
            } /*else if (v.getId() == R.id.two_scroll) {
            Intent intent = new Intent(this, TwoScrollActivity.class);
            startActivity(intent);
        } */
            R.id.nonlift -> {
                val intent = Intent(this, NoLiftHandActivity::class.java)
                startActivity(intent)
            }
            R.id.lift -> {
                val intent = Intent(this, LiftHandActivity::class.java)
                startActivity(intent)
            }
            R.id.conflict -> {
                val intent = Intent(this, ConflictActivity::class.java)
                startActivity(intent)
            }
            R.id.viewpager -> {
                val intent = Intent(this, HeadlineNewsActivity::class.java)
                startActivity(intent)
            }
            R.id.event_dispatch -> {
                val intent = Intent(this, TouchEventDemoActivity::class.java)
                startActivity(intent)
            }
            R.id.outer_intercept -> {
                val intent = Intent(this, OuterInterceptActivity::class.java)
                startActivity(intent)
            }
            R.id.inner_intercept -> {
                val intent = Intent(this, InnerInterceptActivity::class.java)
                startActivity(intent)
            }
            R.id.webview_conflict -> {
                val intent = Intent(this, WebviewActivity::class.java)
                startActivity(intent)
            }
            R.id.x5webview_conflict -> {
                val intent = Intent(this, X5WebviewActivity::class.java)
                startActivity(intent)
            }
        }
    }
}